### PR TITLE
Add example project

### LINKED ISSUE #39: Add a example project
Additional items:
- pubspec.yaml
- pubspec.lock
- main.dart
- analysis_options.yaml
- .gitignore

FYI:
- https://github.com/dart-lang/lints/tree/main/example
- https://github.com/mono0926/pedantic_mono/tree/main/example

### PR DESCRIPTION
**Pull request by blendthink**
## Issue

- close #39 

## Overview (Required)

Add example project.

### CODE REVIEW

**Review by trm11tkr — APPROVED** [review]
LGTM! Approved, but please fix a minor typo before merging.

**Comment by blendthink** [comment]
Thanks!

File: example/pubspec.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+description: An example of yumemi_lints.
+version: 1.0.0
+publish_to: none
+
+environment:
+  sdk: '>=2.17.0 <3.0.0'
+
+dev_dependencies:
+  yumemi_lints:
+    path: ..
```
Comment (Yamasaki-pan961) [inline example/pubspec.yaml:11]: memo-badge

example内でのパッケージ指定は直接Pathを書くのですね
https://github.com/rrousselGit/riverpod/blob/master/examples/counter/pubspec.yaml
https://github.com/imaNNeo/fl_chart/blob/79e6ec7088587007812aef6b194371e4eb807fa3/example/pubspec.yaml#L22

**Review by Yamasaki-pan961 — APPROVED** [review]
LGTMです！

**Pull request merged**